Neos.ContentRepository.Core/Classes/Projection/CatchUp.php

```diff
@@ -104,7 +104,11 @@ public function run(EventStreamInterface $eventStream): SequenceNumber
                 if ($eventEnvelope->sequenceNumber->value <= $highestAppliedSequenceNumber->value) {
                     continue;
                 }
-                ($this->eventHandler)($eventEnvelope);
+                try {
+                    ($this->eventHandler)($eventEnvelope);
+                } catch (\Exception $e) {
+                    throw new \RuntimeException(sprintf('Exception while catching up to sequence number %d', $eventEnvelope->sequenceNumber->value), 1710707311, $e);
```

Comment:
Couldn't this lead to unwanted side effects when catching specific exceptions?

Reply:
Hmm maybe we should create a specific `CatctupException` here but im not aware of any specific exceptions thrown and `catchUpProjection` is never try catched.

Reply:
I meant if a projection were to throw a specific exception we would now "mask" that

Reply:
ha 

>  projection were to throw a specific exception

this is a catchphrase ;) a projection should never throw :D

Reply:
You passed the test 😀